libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.scss

```diff
@@ -1,46 +1,144 @@
+  --sky-override-progress-indicator-status-marker-icon-size: 17px;
+  --sky-override-progress-indicator-status-marker-icon-success-color: #5a9936;
+  --sky-override-progress-indicator-status-marker-line-color: var(
+    --sky-border-color-neutral-medium-dark
+  );
+  --sky-override-progress-indicator-status-marker-line-complete-color: #5a9936;
+  --sky-override-progress-indicator-status-marker-line-width: 1px;
+  --sky-override-progress-indicator-status-marker-margin-right: #{$sky-theme-modern-margin-inline-lg};
+  --sky-override-progress-indicator-status-marker-mode-vertical-top: 3px;
+  --sky-override-progress-indicator-status-marker-status-active-icon-border-color: transparent;
+}
+
+.sky-progress-indicator-status-marker {
   display: flex;
   height: 100%;
 
   &.sky-progress-indicator-status-marker-mode-vertical {
-    flex-basis: $status-marker-width;
-    flex-grow: 0;
     flex-direction: column;
-    margin-right: $status-marker-width;
+    margin-right: var(
+      --sky-override-progress-indicator-status-marker-margin-right,
+      var(--sky-space-gap-label-xl)
+    );
+
+    position: relative;
+    top: var(
+      --sky-override-progress-indicator-status-marker-mode-vertical-top,
+      0
+    );
+    width: var(
+      --sky-override-progress-indicator-mode-vertical-width,
+      max(var(--sky-size-progress_step), var(--sky-size-icon-m))
+    );
 
     .sky-progress-indicator-status-marker-icon {
-      margin-top: $icon-offset;
-      width: $status-marker-width;
+      margin: var(
+          --sky-override-progress-indicator-status-marker-icon-margin-top,
+          0
+        )
+        auto 0;
+      width: var(
+        --sky-override-progress-indicator-status-marker-icon-size,
+        var(--sky-size-progress_step)
+      );
 
       > ::ng-deep sky-icon {
-        margin-left: -1px;
+        margin-left: var(
+          --sky-override-progress-indicator-status-marker-icon-margin-left,
+          0
+        );
+      }
+    }
+
+    &.sky-progress-indicator-status-marker-status-complete {
+      .sky-progress-indicator-status-marker-icon {
+        width: var(
+          --sky-override-progress-indicator-status-marker-icon-size,
+          var(--sky-size-icon-m)
+        );
       }
     }
 
     .sky-progress-indicator-status-marker-line {
       margin: 0 auto;
-      width: 1px;
+      width: var(
+        --sky-override-progress-indicator-status-marker-line-width,
+        var(--sky-border-width-progress_connector)
+      );
     }
   }
 
   &.sky-progress-indicator-status-marker-mode-horizontal {
     flex-direction: row;
     width: 100%;
-    padding-left: 5px;
+    padding-left: var(
+      --sky-override-progress-indicator-status-marker-horizontal-padding-left,
+      0
+    );
 
     .sky-progress-indicator-status-marker-icon {
-      height: $status-marker-width;
+      height: var(
+        --sky-override-progress-indicator-status-marker-horizontal-icon-height,
+        17px
```

Comment:
what is the significance of 17? I'd think it would be 16 if it is a small icon?

Reply:
This is backwards compatibility for wizards. Everything was set to 17px previously.